### PR TITLE
Update FAQ with an entry on cross-browser support

### PR DESCRIPTION
**Pull request by TimvdLippe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -296,6 +296,16 @@ Check out [contributing guide](https://github.com/puppeteer/puppeteer/blob/maste
 The Chrome DevTools team maintains the library, but we'd love your help and expertise on the project!
 See [Contributing](https://github.com/puppeteer/puppeteer/blob/master/CONTRIBUTING.md).
 
+#### Q: What is the status of cross-browser support
```
Comment (mathiasbynens) [inline README.md]:
```suggestion
#### Q: What is the status of cross-browser support?

```

Comment (mathiasbynens) [reply README.md]: also please insert a newline here

**Review by mathiasbynens — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -296,6 +296,17 @@ Check out [contributing guide](https://github.com/puppeteer/puppeteer/blob/maste
 The Chrome DevTools team maintains the library, but we'd love your help and expertise on the project!
 See [Contributing](https://github.com/puppeteer/puppeteer/blob/master/CONTRIBUTING.md).
 
+#### Q: What is the status of cross-browser support?
+
+Historically, Puppeteer supported Firefox indirectly through puppeteer-firefox, which relied on a custom, patched version of Firefox. This approach was also known as “Juggler”.
+After discussions with Mozilla, we collectively concluded that relying on custom patches was infeasible.
+Since then, we have been collaborating with Mozilla on supporting Puppeteer on “stock” Firefox.
+From Puppeteer v2.1.0 onwards, as an experimental feature, you can specify [`puppeteer.launch({product: 'firefox'})`](https://github.com/puppeteer/puppeteer/blob/v2.1.0/docs/api.md#puppeteerlaunchoptions) to run your Puppeteer scripts in Firefox, without any additional custom patches.
```
Comment (rodoabad) [inline README.md:304]: This doesn't specify if consumers still need `puppeteer-firefox` or not.

`puppeteer` doesn't come with Firefox by default so setting it to `product: 'firefox'` is only the first step.

Comment (rodoabad) [reply README.md:304]: @mathiasbynens @TimvdLippe

**Pull request merged**